core-services/image-mirroring/openshift/mapping_origin_4_17

```diff
@@ -1,5 +1,4 @@
 registry.ci.openshift.org/origin/4.17:baremetal-installer quay.io/openshift/origin-baremetal-installer:4.17 quay.io/openshift/origin-baremetal-installer:4.17.0 quay.io/openshift/origin-baremetal-installer:latest
-registry.ci.openshift.org/origin/4.17:cluster-samples-operator quay.io/openshift/origin-cluster-samples-operator:4.17 quay.io/openshift/origin-cluster-samples-operator:4.17.0 quay.io/openshift/origin-cluster-samples-operator:latest
```

Comment:
i would recommend that once you have verified that the jobs pass and the image is created in registry.ci, i.e `registry.ci.openshift.org/origin/scos-4.17:cluster-samples-operator`, you can add the mirroring stanza

Reply:
Since this file is autogenerated, how do I reference the scos image here?